lib/linter/eslint.rb

```diff
@@ -0,0 +1,45 @@
+module Linter
+  class ESLint < Base
+    private
+
+    def config_files
+      [".eslintrc.js", ".eslintrc.json"]
+    end
+
+    def linter_executable
+      'eslint .'
+    end
+
+    def options
+      {:format => 'json'}
+    end
+
+    def filtered_files(files)
+      files.select do |file|
+        file.end_with? '.js'
+      end
+    end
+
+    def convert_parsed(array)
+      files = array.map { |f| convert_file(f) }
+
+      {:files   => files,
+       :summary => {:offense_count     => files.reduce(0) { |memo, f| memo + f.offenses.count },
```

Comment:
I believe `files.reduce(0) { |memo, f| memo + f.offenses.count }` can be simplified to `files.map { |f| f.offenses.count }.reduce(:+)`